inst/templates/ad_adpc.R

```diff
@@ -41,18 +41,6 @@ param_lookup <- tibble::tribble(
   "DOSE", "DOSE", "Xanomeline Patch Dose", 2,
 )
 
-# ---- User defined functions ----
-
-# Here is an example of how you can create your own function that
-# operates on vectors, which can be used in `mutate`.
-format_avalcat1n <- function(param, aval) {
```

Comment:
I removed this as this is not used in the template. Should I have left this in? 
@bundfussr

Reply:
In my opinion it could be removed.

@jeffreyad , any objections?